coursemapper-kg/concept-map/src/data.py

```
@@ -1,12 +1,13 @@
 import re
 from typing import List, Callable
 from numpy import ndarray
+from typing import Optional
 
 def clean_id(id: str) -> str:
     return re.sub(r'[^a-zA-Z0-9_]', '_', id)
 
 class Node:
-    def __init__(self, id: str, name: str, uri: str, type: str, weight: float, wikipedia: str, text: str, is_draft: bool, embedding: ndarray | None = None, keyphrases: List[str] = [], embedding_model: str | None = None):
+    def __init__(self, id: str, name: str, uri: str, type: str, weight: float, wikipedia: str, text: str, is_draft: bool, isNew: Optional[bool] = None, isEditing:Optional[bool] = None, embedding: ndarray | None = None, keyphrases: List[str] = [], embedding_model: str | None = None):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)